dpctl/_sycl_context.pyx

```diff
@@ -125,20 +125,22 @@ cdef class SyclContext(_SyclContext):
                 # Create a CPU device using the opencl driver
                 cpu_d = dpctl.SyclDevice("opencl:cpu")
                 # Partition the CPU device into sub-devices, each with two cores.
-                sub_devices = create_sub_devices(partition=2)
+                sub_devices = cpu_d.create_sub_devices(partition=2)
                 # Create a context common to all the sub-devices.
                 ctx = dpctl.SyclContext(sub_devices)
                 assert(len(ctx.get_devices) == len(sub_devices))
 
-        - Invoking the constuctor with a named ``PyCapsule`` with the name
-          **SyclContextRef** that carries a pointer to a ``sycl::context``
-          object.
+        - Invoking the constuctor with a named ``PyCapsule`` with name
+          **"SyclContextRef"** that carries a pointer to a ``sycl::context``
+          object. The capsule will be renamed upon successful consumption
+          to ensure one-time use. A new named capsule can be constructed by
+          using `ctx._get_capsule()` method.
```

Comment:
How about :func:`dpctl.SyclContext._get_capsule()`? We will get a link to the doc for `_get_capsule` instead of italicizing the function name.

Reply:
done